### PR TITLE
Fix Android build for branch python3

### PR DESCRIPTION
**Pull request by agilewalker**
Also need a fix for python-for-android. https://github.com/agilewalker/python-for-android/commit/4f7aee6d6d287196ecc499b3df25fd91870909ee which is part of pull request https://github.com/kivy/python-for-android/pull/1195

### CODE REVIEW

File: contrib/make_packages [hunk 1]
```diff
@@ -5,8 +5,8 @@ contrib=$(dirname "$0")
 whereis pip3
 if [ $? -ne 0 ] ; then echo "Install pip3" ; exit ; fi
 
-rm $contrib/packages/ -r
```
Comment (bauerj) [inline contrib/make_packages]: This was there to allow running the script from somewhere else.

Comment (agilewalker) [reply contrib/make_packages]: Could you explain where exactly? Have doubts about this change, but there seems to be no alternative other than running contrib/make_packages on project root directory.

Comment (bauerj) [reply contrib/make_packages]: Anywhere, I guess. `$contrib` is set to the absolute path of the contrib directory. This enables you to call `make_packages` from e.g. `/root/` without having `/root/packages` removed.

Comment (bauerj) [reply contrib/make_packages]: Not saying that this is strictly necessary, but if you remove it you should give your reason to do so and remove the line where that variable is set above.

Comment (agilewalker) [reply contrib/make_packages]: Sorry, brain fart. Meant ./electron-cash which seems looking for ./packages/ in the root project directory.
https://github.com/fyookball/electrum/blob/200ccfaad1dd87f58ec7aec2fec6b43c839765ed/electron-cash#L52-L53

Comment (agilewalker) [reply contrib/make_packages]: $contrib is still needed for https://github.com/agilewalker/electrum/blob/e5103a235cbfa868b6c500c78696eb54c0e9d93d/contrib/make_packages#L11

Comment (bauerj) [reply contrib/make_packages]: Okay, that makes sense :+1: Thanks for clarifying.

**Comment by agilewalker** [comment]
With a recent change https://github.com/kivy/python-for-android/pull/1195/commits/3aa79e523f4435ac47e58100d8291e7a0f43fa19, it is now possible to build on python3.6.

**Comment by bauerj** [comment]
Does that make this PR obsolete?

**Comment by agilewalker** [comment]
No, it still works. Now if you want a Python 3.6 build, you can make a single line change of
https://github.com/agilewalker/electrum/blob/caac100d8d69b762da92f463e322e19fcae0935a/gui/kivy/tools/buildozer.spec#L34
with
`requirements = python3crystax==3.6, android, openssl, plyer, kivy==master`.
Voila! You are now ready to build.

**Pull request closed without merging**